packages/backend/src/modules/dashboard/dashboard.repository.ts

```diff
@@ -1,27 +1,50 @@
-import { CreateTaskCommandDTO } from 'shared'
+import { ObjectId } from 'mongoose'
+import { CreateTaskCommandDTO, UpdateTaskCommandDTO } from 'shared'
 import DashboardModel from './Dashboard'
 import { Task } from '../task/task'
 
 export class DashboardRepository {
   private readonly _dashboardModel = DashboardModel
 
+  // findIndexInDatabase is looking for index by id in an Array of Objects
+  private findIndexOfDocument(id: string, array: { _id: ObjectId }[]): number {
+    const columnOrTaskIndex = array.findIndex(
+      (columnOrTaskID: { _id: ObjectId }) => String(columnOrTaskID._id) === id
+    )
+    return columnOrTaskIndex
```

Comment:
don.t do code like that. Read about SCR create another metod for task. Be precise

Reply:
https://en.wikipedia.org/wiki/Single-responsibility_principle#:~:text=The%20single%2Dresponsibility%20principle%20(SRP,it%20should%20encapsulate%20that%20part.

Reply:
Return name would be confusing but this function has single responsibility. I changed this name.